lib/tasks/mfa_policy.rake

```diff
@@ -51,4 +51,23 @@ namespace :mfa_policy do
       print format("\r%.2f%% (%d/%d) complete", i.to_f / total_users * 100.0, i, total_users)
     end
   end
+
+  # This task is meant to be run once MFA Phase 3 has launched - send out on Aug 15, 2022
+  # For more information on the MFA Phase 3 rollout, refer to this RFC:
+  # https://github.com/rubygems/rfcs/pull/36/files#diff-3d5cc3acc06fe7e9150fdbfc43399c5ad42572c122187774bfc3a4857df524f1R69-R85
+  # rake mfa_policy:announce_enforcement_for_popular_gems
+  desc "Send email to notify users that MFA is now being enforced due to MFA Phase 3 rollout"
+  task announce_enforcement_for_popular_gems: :environment do
+    # users who own at least one gem with 180,000,000 downloads or more, with weak MFA or no MFA enabled
+    users = User.joins(rubygems: :gem_download).where("gem_downloads.count >= 180000000").where(mfa_level: %w[disabled ui_only]).uniq
+    total_users = users.count
+    puts "Sending #{total_users} MFA required for popular gems email"
+
+    i = 0
+    users.each do |user|
+      Mailer.delay.mfa_required_popular_gems_announcement(user.id) if mx_exists?(user.email)
```

Comment:
What do we do if `mx_exists?` is false?

Reply:
Great q Jacques! Currently, nothing. I think this may be similar to some [feedback](https://github.com/rubygems/rubygems.org/pull/3166#discussion_r937773263) you noted in a previous mailer ... that maybe we _can_ or _should_ do something in the `false` case? 

For instance, printing out those emails so we have visibility into those accounts. Ultimately, I had held off adding it last time because I was [waiting to hear](https://github.com/rubygems/rubygems.org/pull/3166#discussion_r938414315) whether whoever's running the rake task would action on that info. 

If not, it'd just be noise in the logs. 

That said, I'm happy to just move forward with adding that logging if you think we should do it anyway (whether the info will be used or not).

Reply:
My gut instinct is to log, Just In Case (tm)

Reply:
No problemo! Can do 💪.

Reply:
Done! It's been [updated](https://github.com/rubygems/rubygems.org/pull/3171/files#diff-ca4ffd6e42af383a715c6c72f563969227bc4c41dc40907a0a90a25f763dabccR75-R80) 😊. Thanks Jacques!